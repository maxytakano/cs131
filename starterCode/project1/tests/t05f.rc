float x;

function: float _foo(bool x) {
    return x;
}

function: int main() { 
    float x = _foo();
}
