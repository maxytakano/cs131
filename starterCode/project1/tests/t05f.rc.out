Error, "/tmp/cs131_testrunner_M8mrMF/file.rc": 
  Type of return expression (bool), not assignment compatible with function's return type (float).
Error, "/tmp/cs131_testrunner_M8mrMF/file.rc": 
  Number of arguments (0) differs from number of parameters (1).
Error, "/tmp/cs131_testrunner_M8mrMF/file.rc": 
  Return required in function, none found.
Compile: failure.
